robocup_ws/src/Planner/src/striker_controller.py

```diff
@@ -2,26 +2,59 @@
 import numpy as np
 from BasicCommonActions.go_to_action import simple_go_to_action
 from game_interfaces.msg import Position
+from BasicCommonActions.go_to_action import go_to_fast
+from Planner.src.brachistochrone import linear, parabola
+import math
 
 
 class Team1StrikerController:  #(Robot)
```

Comment:
Probably we should rename it -> this is probably no longer Team 1 specific class